README.md

```diff
@@ -176,10 +176,14 @@ additionalSigScopes: {}
 Running `setconf` with no filename will set the above default options.
 
 #### step
-The `step [n]` command performs n state transitions from the current execution state, ending at one of the valid states for a length (current + n) state traversal from the initial state.
+The `step [n | constraints]` command performs n state transitions from the current execution state, ending at one of the valid states for a length (current + n) state traversal from the initial state.
 
 Specify an integer value of n >= 1. By default, n = 1.
 
+A list of constraints may also be specified, as a comma-separated list enclosed by square brackets.
```

Comment:
You should be more clear that the user can either specify the number of steps **or** a list of constraints. Something like: `"Instead of a numeric number of steps, step constraints may be specified as a comma-separated list enclosed in square brackets."`